src/components/header/Header/Header.stories.tsx

```diff
@@ -0,0 +1,345 @@
+import React, { useState } from 'react'
+import { Header } from './Header'
+
+import { Title } from '../Title/Title'
+import { PrimaryNav } from '../PrimaryNav/PrimaryNav'
+import { Search } from '../../Search/Search'
+import { Menu } from '../Menu/Menu'
+import { MegaMenu } from '../MegaMenu/MegaMenu'
+import { NavButton } from '../NavButton/NavButton'
+import { NavDropDown } from '../NavDropDown/NavDropDown'
+import { ExtendedNav } from '../ExtendedNav/ExtendedNav'
+
+export default {
+  title: 'Header/Header',
+  parameters: {
+    info: `
+USWDS 2.0 Header component
+
+Source: https://designsystem.digital.gov/components/header/
+`,
+  },
+}
+
+const mockSubmit = (): void => {
+  /* mock submit fn */
+}
+
+export const BasicHeader = (): React.ReactElement => {
+  const [expanded, setExpanded] = useState(false)
+  const onClick = (): void => setExpanded((prvExpanded) => !prvExpanded)
+
+  const testMenuItems = [
+    <a href="#linkOne" key="one">
+      Current link
+    </a>,
+    <a href="#linkTwo" key="two">
+      Simple link Two
+    </a>,
+  ]
+
+  const [isOpen, setIsOpen] = useState([false, false])
+
+  const onToggle = (index: number): void => {
+    setIsOpen((prevIsOpen) => {
+      const newIsOpen = Array(prevIsOpen.length).fill('false')
```

Comment:
Just caught this while testing.. this needs to be `.fill(false)` (boolean), since `'false'` the string evaluates to truthy

Reply:
Good catch! thank you